lib/src/room.dart

```diff
@@ -1560,14 +1560,16 @@ class Room {
   /// [[Membership.join, Membership.invite, Membership.knock]]
   /// Set [cache] to `false` if you do not want to cache the users in memory
   /// for this session which is highly recommended for large public rooms.
-  Future<List<User>> requestParticipants(
-      [List<Membership> membershipFilter = const [
-        Membership.join,
-        Membership.invite,
-        Membership.knock,
-      ],
-      bool suppressWarning = false,
-      bool cache = true]) async {
+  Future<List<User>> requestParticipants({
+    List<Membership> membershipFilter = const [
+      Membership.join,
+      Membership.invite,
+      Membership.knock,
+    ],
+    bool suppressWarning = false,
+    bool cache = true,
```

Comment:
if this is a breaking change, can you also be more verbose with the `cache` parameter? i.e change it to `cacheInMemory` for example?

Reply:
Good idea. Have added it :)